src/psyclone/tests/psyir/transformations/transformations_test.py

```diff
@@ -62,6 +63,29 @@
                                 "gocean1p0")
 
 
+@pytest.fixture
```

Comment:
Please use the `name=sample_psyir` feature and rename the routine itself as `sample_psyir_fixture`. This will prevent pylint warnings about `sample_psyir` being redefined.

Reply:
Done